subprojects/docs/src/samples/build-organization/structuring-software-projects/README.adoc

```diff
@@ -14,3 +14,5 @@ There are different ways to work with the sample:
   For example, `cd server-application` and run the app using  `../gradlew :app:bootRun`.
 - You may only build or import a selected component (and its dependencies).
   For example, only import the `user-feature` build in the IDE.
+
+// TODO:Finalize Upload Removal - Issue #21439
```

Comment:
Is this because the sample is using Spring?  If we leave in the Upload task for 8.0, this should mean we can release without upgrading Spring, right?

Reply:
Right, this is just so we can easily find our way back to all the changes that will be needed when we do finally remove the `Upload` task.  With the Spring releases that have happened already and are happening next week, we should be able to clear some of these.